changelog.d/9084.bugfix

```diff
@@ -0,0 +1 @@
+Don't blacklist connections to the configured proxy. Contributed by @Bubu.
```

Comment:
FTR this should have a) started with `Fix...` and b) stated what version it regressed in, sorry for not picking it up at the time :)